infra/app/storage.bicep

```diff
@@ -0,0 +1,26 @@
+param name string
+param location string = resourceGroup().location
+param allowBlobPublicAccess bool = true
+param tags object = {}
+var serviceName = 'storage'
+
+module storage '../core/storage/storage-account.bicep' = {
+  name: '${name}-storage-module'
+  params: {
+    name: name
+    location: location
+    tags: union(tags, { 'azd-service-name': serviceName })
+    allowBlobPublicAccess: allowBlobPublicAccess
+  }
+}
+
+output name string = storage.outputs.name
+output id string = storage.outputs.id
+#disable-next-line outputs-should-not-contain-secrets
+output primaryKey string = storage.outputs.primaryKey
+output primaryEndpoints object = storage.outputs.primaryEndpoints
```

Comment:
Let's figure out a way to mod the architecture to now have to output keys and connection strings.  I'm thinking we introduce a key vault.